migrations/Version20241213092228.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20241213092228 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add user_id to appointment and fix foreign keys';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE appointment DROP FOREIGN KEY FK_FE38F844C3C6F69F');
+        
+        $this->addSql('ALTER TABLE appointment ADD user_id INT NOT NULL, CHANGE car_id car_id INT NOT NULL, CHANGE appointment_type appointment_type ENUM("maintenance", "registration", "polishing", "painting") NOT NULL');
+        
+        $this->addSql('ALTER TABLE appointment ADD CONSTRAINT FK_FE38F844A76ED395 FOREIGN KEY (user_id) REFERENCES `users` (id)');
+        $this->addSql('ALTER TABLE appointment ADD CONSTRAINT FK_FE38F844C3C6F69F FOREIGN KEY (car_id) REFERENCES cars (id)');
+        
+        $this->addSql('CREATE INDEX IDX_FE38F844A76ED395 ON appointment (user_id)');
```

Comment:
make only one migration for this table, since it's in ealry phases of development, to make it clean

Reply:
have in mind that you will probably need to redo database migration, since you will have 2 migrations executed that doesn't exist